docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -0,0 +1,92 @@
+# Feature Band Support: Work Plan
+
+This document builds on top of [Managing SDK Bands](https://github.com/dotnet/dotnet/blob/main/docs/VMR-Managing-SDK-Bands.md) with a concrete work plan for providing feature band support in the VMR. Please read that doc as a prerequisite.
+
+Document convention:
+* There can be a variable number of feature band releases (e.g. 10.0.1xx, 10.0.2xx, 10.0.3xx, etc). For purposes of this document, we mainly care about the distinction between the 1xx branch and all the other non-1xx feature band branches. For simplicity, 2xx will be used as a notation to mean any non-1xx feature band branch.
+
+## Plan Details
+
+### Branch Management
+
+The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation.
```

Comment:
Right now I'm on the side of avoiding subsetting the sources. Just being able to branch the repo and create a 2xx or 3xx branch, without deleting a bunch of files, probably has a ton of value. It keeps the shared history and means you can merge from 2xx back to 1xx if you wanted to.